packages/eslint-config-custom/index.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     jest: true,
     node: true
   },
-  extends: ['next', 'turbo', 'prettier'],
```

Comment:
Wasn't this here to ensure ESLint does not collide with its stylistic rules with Prettier?